resources/views/transfers/show.blade.php

```diff
@@ -204,7 +204,7 @@
     <div class="col-xs-12 actions">
       <a class="btn btn-sm btn-secondary" role="button" href="{{ route('transfers.edit', $transfer->id) }}"><i class="fa fa-pencil" aria-hidden="true"></i> Edit</a>
       @if ($transfer->cut === null && $transfer->type === 'Audio') 
-      <a class="btn btn-sm btn-secondary" role="button" href="{{ route('masters.cuts.create', array('masterId' => $transfer->preservationMaster->id, 'transferId' => $transfer->id)) }}"><i class="fa fa-plus" aria-hidden="true"></i> Add Cut</a>
+      <a class="btn btn-sm btn-secondary" role="button" href="{{ route('masters.cuts.create', array('masters' => $transfer->preservationMaster->id, 'transferId' => $transfer->id)) }}"><i class="fa fa-plus" aria-hidden="true"></i> Add Cut</a>
```

Comment:
Why was this change needed?

Reply:
Ah! This was to fix a bug that @ekemeyer noted in the [ticket](https://jira.lib.unc.edu/browse/APPDEV-9849). The logged error was:
`Missing required parameter for [Route: masters.cuts.create]: masters`
I think that it now looks for a more standardized convention when sending in an array of params.

Reply:
Ah OK, thanks!